packages/metadata/src/decorator-factory.ts

```diff
@@ -9,7 +9,9 @@ import * as debugModule from 'debug';
 import {MetadataMap, DecoratorType, MetadataKey} from './types';
 const debug = debugModule('loopback:metadata:decorator');
 
-// tslint:disable:no-any
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Where is `ban-types` rule coming from? It is not enabled in our current tslint config.

Reply:
This rule actually helps us catch a few issues, such as:

```ts
let str: String; // it should be `let str: string`
```

Reply:
I see. I don't understand the nuance between `String` and `string`, but I trust the TypeScript & eslint teams that it's a good idea to use `string` instead of `String`.

As I wrote elsewhere, this patch is already too large, let's not make it even larger by enabling extra rules.

I would like to keep this patch focused on the migration from TSLint to ESLint only, and leave out additional rules for follow pull requests. I have my own list of rules to enable too :)

Reply:
> I would like to keep this patch focused on the migration from TSLint to ESLint only

Based on your comment below, I better understand your motivation here and don't mind to enable ban-types rule as part of this PR.